### PR TITLE
add instructions doc

### PR DESCRIPTION
**Pull request by JuanCaicedo**
Adds a doc with instructions about NodeSchool! My thought is that we can have these printed out and give them to people as they walk through the door, so that they can get started as soon as they get there instead of waiting for an official start. I'm hoping it'll also help us get people pairing :)

@robbiethegeek @danpaz @joshfinnie and maybe you'll have some thoughts @jasonrhodes

### CODE REVIEW

**Comment by danpaz** [comment]
Looks great to me!

File: instructions.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Getting Started
+
+- Go to `www.nodejs.org` and download either v4 or v5.
+- Get a text editor if you don't already have one. The following are all free
+  and recommended:
+  * Atom
+  * Sublime Text 2
+  * Brackets
+  * Textmate
```
Comment (robbiethegeek) [inline instructions.md]: Let's remove Textmate, it isn't cross os and add VS Code.

File: instructions.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Tips for doing the workshops
+
+- Read all of the instructions! It's easy to miss parts
+- If you want to know what's going on with your code, use
+`console.log('myData', myData)`. It will first print the string `myData` to
+the screen and then the value of the variable `myData`
+- Ask questions! Especially when you get to working with callbacks or modules.
+Raise your hand at any time or get one of the mentor's attention. We're glad to
+help :)
```
Comment (robbiethegeek) [inline instructions.md:41]: Call out the name-of-workshop run as a way to troubleshoot, and name-of-workshop verify to confirm DO NOT use node my-program.js my made up params

**Comment by JuanCaicedo** [comment]
@joshfinnie or @robbiethegeek I don't have commit rights to this repo to merge this PR. Could you add the NodeDC organization so we can all have access?

**Pull request merged**